tms_objects2.erb

```diff
@@ -274,6 +274,16 @@ xsi:schemaLocation="http://www.lido-schema.org http://www.lido-schema.org/schema
         <% end %>  
         <% end %>   <%# ConAltNames |a| %>
                   </lido:nameActorSet>
+                  <lido:nationalityActor>
+        <% ConAttributes.each do |b| %>
+        <% if b["ConstituentID"] == c["ConstituentID"] and b["thesxreftype"] != "sourceTermID" %>
+                    <lido:conceptID lido:source="<%= b["UCTS"] %>" lido:type="<%= b["thesxreftype"] %>" lido:label="<%= b["Term"] %>"><%= b["FinalST"] %></lido:conceptID>
```

Comment:
b["FinalST"] should be b["SourceTermID"]

for example the conceptID should be "7008653" not "tgn7008653"